corehq/apps/hqcase/utils.py

```diff
@@ -50,6 +50,8 @@ def submit_case_blocks(case_blocks, domain, username="system", user_id=None,
     make it easier to trace the source. All new code should use this
     argument. A human recognizable value is recommended outside of test
     code. Example: "auto-close-rule-<GUID>"
+    :param max_wait: Maximum time to allow the process to be delayed if
+    the project is over its submission rate limit.
```

Comment:
It would be useful if this docstring explained the meaning of the default value since `...` or `Ellipsis` does not make that obvious. It could be as simple as pointing to the dos for `submit_form_locally`.